agent/bench-scripts/tests/pbench-linpack/test-66.post

```diff
@@ -0,0 +1 @@
+test-64.post
```

Comment:
Since `agent/bench-scripts/tests/pbench-linpack/test-66.pre` references `test-65.pre`, shouldn't this reference `test-65.post`?  (It would be nice to keep things from getting _too_ tangled....)

Reply:
[This has been resolved; I don't see any way to mark it as such on this page, though.]